oraibtcbridge/assetlist.json

```diff
@@ -0,0 +1,62 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "oraibtcbridge",
+  "assets": [
+    {
+      "description": "The main native staking token of Orai Bitcoin Bridge",
+      "denom_units": [
+        {
+          "denom": "uoraibtc",
+          "exponent": 0
+        },
+        {
+          "denom": "ORAIBTC",
+          "exponent": 6
+        }
+      ],
+      "base": "uoraibtc",
+      "name": "oraibtcbridge",
+      "display": "ORAIBTC",
+      "symbol": "ORAIBTC",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/orai-white.png",
```

Comment:
It should be using the same logo as ORAI token?
In which case, it might make sense to use image sync, so that any updates to the image will update everywhere without anything (like this reference) breaking.